### PR TITLE
Various fixes.

### PR DESCRIPTION
**Pull request by Fraser999**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/maidsafe_vault/121)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @maqi

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/main.rs [hunk 1]
```diff
@@ -90,12 +90,12 @@ impl Vault {
 // ==========================   Program Options   =================================
 static USAGE: &'static str = "
 Usage:
-  maidsafe_vault (--first | --node [<peer>...])
+  maidsafe_vault [<peer>...]
```
Comment (benjaminbollen) [inline src/main.rs]: Needs an update to the executable_test integration test; it calls the nodes with -n and fails

**Comment by dirvine** [comment]
Reviewed 3 of 3 files at r1, 1 of 1 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/121)

**Pull request merged**